### PR TITLE
Basin bot updates

### PR DESCRIPTION
**Pull request by soilking**
For Discord/Telegram
- Add event handler for `BoreWell`
- Add a new channel for all swap and liquidity events for all other wells (that aren't whitelisted)

After deploying, the #other-wells channel should be made public in the Basin discord.

Fixes
- Fix broken daily report and liquidity amount bot

### CODE REVIEW

File: data_access/eth_chain.py [hunk 1]
```diff
@@ -1047,25 +1044,21 @@ def __init__(self, event_client_type, address=""):
...
             self._contract_addresses = [BEANSTALK_ADDR]
             self._events_dict = MARKET_EVENT_MAP
             self._signature_list = MARKET_SIGNATURES_LIST
-            self._set_filters()
         elif self._event_client_type == EventClientType.BARN_RAISE:
             self._contracts = [get_fertilizer_contract(self._web3)]
             self._contract_addresses = [FERTILIZER_ADDR]
             self._events_dict = FERTILIZER_EVENT_MAP
             self._signature_list = FERTILIZER_SIGNATURES_LIST
-            self._set_filters()
```
Comment (funderbrker) [inline data_access/eth_chain.py:1062]: why did you remove these calls to reset the filters? these were put in for edge cases where the filters change during runtime.

File: README.md [hunk 2]
```diff
@@ -8,9 +8,13 @@ Included in this repo is a set of bots that disseminate information to Beanstalk
...
 Environment variables necessary:
-- `ETH_CHAIN_API_KEY`
-- `SUBGRAPH_API_KEY`
+- `ALCHEMY_ETH_API_KEY`
 - `DISCORD_BOT_TOKEN` (`DISCORD_BOT_TOKEN_PROD` for prod application)
-- `TELE_BOT_KEY` (`TELEGRAM_BOT_TOKEN_PROD` for prod application)
+- `DISCORD_BASIN_BOT_TOKEN` (`DISCORD_BASIN_BOT_TOKEN_PROD` for prod application)
+- (`TELEGRAM_BOT_TOKEN_PROD` for prod application)
+
+Create an `env.dev` file using the provided example and place your varaibles there. Then, execute `./dev.sh <module>`. For example, to run the main set of bots, execute `./dev.sh bots.discord_bot`.
```
Comment (funderbrker) [inline README.md:20]: what is the reason for changing the handling of tokens for these three tokens and have they been set up to function on the server prod instances using systemd?

File: data_access/eth_chain.py (hunk 1 above)
Comment (soilking) [reply data_access/eth_chain.py:1062]: It was moved to the end of the if/else ladder instead of including in each condition

File: README.md (hunk 2 above)
Comment (soilking) [reply README.md:20]: The only thing I changed was with respect to local development. Instead of having to export the required variables globally or in the session that starts the bots, it gets taken care of in that wrapper script I created, which pulls those variables from another file
